students/ian_letourneau/Lesson08/test_circle_lab.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+# Ian Letourneau
+# 6/4/2018
+
+import math
+import circle_lab as cr
```

Comment:
note that aliases should be generally avoided as they reduce readability 

you can instead do:
`from circle_lab import Circle`